### PR TITLE
Add week 2 slides

### PR DESCRIPTION
**Pull request by jgkamat**
Here are the slides for week 2. Diff after the first commit to see the new changes. Addresses #11

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -3,9 +3,9 @@ RoboCup Software Training (Created in 2017)
 
 To Navigate Slides, Press ~space~ for going forward and ~p~ for going back. Press ~?~ for help!
 * Slides
-1. [[https://robojackets.github.io/robocup-training/slides/1][Training Meeting 1]]
+1. [[https://robojackets.github.io/robocup-training/slides/2017/1][Training Meeting 1]]
 
-* Docs
+* TODO Docs
```
Comment (jgkamat) [inline README.org]: x

**Comment by jgkamat** [comment]
@joshhting could you give this a quick look over, since it's needed tomorrow?

**Comment by JNeiger** [comment]
Does the readme need to be updated to include a link to this?

**Comment by jgkamat** [comment]
Ah, yup, I'll commit that to master now.

**Comment by joshhting** [comment]
I did that already...

**Pull request merged**